app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -1304,28 +1427,35 @@ private void displayMessage(String message, boolean bad, boolean suppressToast)
             Toast.makeText(this, message, Toast.LENGTH_LONG).show();
         }
 
-        TextView syncMessage = (TextView)findViewById(R.id.home_sync_message);
-        
-        syncMessage.setText(message);
-
-
-        //Need to transplant the padding due to background affecting it
-        int[] padding = {syncMessage.getPaddingLeft(), syncMessage.getPaddingTop(), syncMessage.getPaddingRight(),syncMessage.getPaddingBottom() };
-        if(bad){
-            syncMessage.setTextColor(getResources().getColor(R.color.red));
-            syncMessage.setTypeface(null, Typeface.BOLD);
-            syncMessage.setBackgroundDrawable(getResources().getDrawable(R.drawable.bubble_danger));
-        }
-        else{
-            syncMessage.setTextColor(getResources().getColor(R.color.black));
-            syncMessage.setTypeface(null, Typeface.NORMAL);
-            syncMessage.setBackgroundDrawable(getResources().getDrawable(R.drawable.bubble));
+        if(isUsingNewUI()) {
+            adapter.setNotificationTextForButton(R.layout.home_sync_button, false, message);
+//            int margin = getResources().getDimensionPixelSize(R.dimen.margin);
```

Comment:
Please leave a comment describing why this code is commented out. Commented-out code often gets left in place because people have no way of knowing how important/useful it might be.